app/cp_nigeria/helpers.py

```diff
@@ -736,14 +736,17 @@ def collect_financial_params(self):
             "grant_share",
             "debt_interest_MG",
             "debt_interest_SHS",
+            "loan_maturity",
+            "grace_period",
             "equity_interest_MG",
             "equity_interest_SHS",
             "equity_community_amount",
             "equity_developer_amount",
         )
         qs_ed = EconomicData.objects.filter(project=self.project).values("discount", "tax")
-        financial_params = pd.concat([pd.DataFrame.from_records(qs_eq), pd.DataFrame.from_records(qs_ed)], axis=1)
+        financial_params = qs_ed.first() | qs_eq.first()
```

Comment:
A comment to what this command does would be welcome :)

Reply:
That is the merge operator for dictionaries; it is basically doing the same as the deleted line (concatenating the EconomicData values with the EquityData values), but on a dictionary instead of a dataframe. I decided to change this because it was a bit silly to have a dataframe with only one row.